stadgar/body.md

```diff
@@ -196,6 +196,8 @@ Om minst 30 sektionsmedlemmar, sektionsrevisor enligt §[8.1] eller THS styrelse
 
 Kallelse ska innehålla information om en reservtid och lokal där mötet återupptas om ajournering beslutas enligt §[3.12]. Reservtiden ska vara senast 7 dagar efter den ordinarie tiden för mötet.
```

Comment:
This line should be removed, no duplicate paragraphs. 


```suggestion
```

Reply:
I was a bit too hasty